ci-operator/step-registry/telcov10n/pre-ga-catalog/telcov10n-pre-ga-catalog-commands.sh

```
@@ -495,10 +249,62 @@ EOF
+apiVersion: operators.coreos.com/v1alpha1
+kind: CatalogSource
+metadata:
+  name: redhat-operators-custom-4-21
+  namespace: openshift-marketplace
+spec:
+  image: quay.io/prega/prega-operator-index:v$1
+  displayName: Red Hat Custom Operators Catalog for $1
+  sourceType: grpc
+EOF
+)
+  export imageDigestMirrorSet=$(cat <<EOF
```

Shellcheck (0.11.0)

[info] The mentioned syntax error was in this variable assignment.

(SC1009)